src/gwt/src/org/rstudio/studio/client/workbench/UserInterfaceHighlighter.java

```diff
@@ -295,9 +416,11 @@ private void repositionHighlighters()
    // Private members ----
    
    private JsVector<HighlightQuery> highlightQueries_;
+   private ArrayList<Boolean> queryCallbackStatuses_;
```

Comment:
nit: best practice is to use the Java interface type (`List`) and then instantiate it with the concrete class as required (`ArrayList`)

Reply:
Thanks, fixed!